docs/user/integration/dynatrace/secret-example.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dynakube
+  namespace: dynatrace
+stringData:
+  # Dynatrace endpoint URL including /api, like https://123.live.dynatrace.com/api
+  apiurl: "<Dynatrace Endpoint>"
```

Comment:
Please validate if `apiurl` (all lowercase) is consistently used in the other documentation.